### ISSUE TITLE
Doc: Subclasses in MemberIndex found under idx.instance.class

### ISSUE THREAD

**Issue by michael** [substance/substance#229]
![image](https://cloud.githubusercontent.com/assets/2931/11147819/e922b2d0-8a18-11e5-9767-5e61f4a9eaf2.png)

Shouldn't those show up under idx.class.property or idx.class.class ? Currently they are annotated like this.

``` js
/** 
  Virtual Components. An abstract class for Virtual components, created via Component.$$.

  @class Component.VirtualNode
  @abstract
*/

function VirtualNode() {
}
```

I also tried to supply @static explicitly and use a fully qualified name like so:

``` js
/** 
  Memberclass Foo

  @class Component.MemberClass
  @static
  @abstract
*/
Component.MemberClass = function() {
};
```

But still it's found under idx.instance.class. 

I can implement the MemberIndexComponent so that it looks for nested classes under that url, but maybe it's easy change it so they show up under class.class or class.property.

**Comment by obuchtala** [comment]
Yep, u r right.

**Comment by obuchtala** [comment]
Will fix...

**Comment by obuchtala** [comment]
Fixed via b8da605efc7e511ba362a63acde6251eafe6a363

**Comment by michael** [comment]
Thanks! Works.
